pkg/security/probe/probe_ebpfless.go

```diff
@@ -108,14 +108,15 @@ func (p *EBPFLessProbe) handleClientMsg(cl *client, msg *ebpfless.Message) {
 func copyFileAttributes(src *ebpfless.FileSyscallMsg, dst *model.FileEvent) {
 	if strings.HasPrefix(src.Filename, "memfd:") {
 		dst.PathnameStr = ""
-		dst.BasenameStr = src.Filename
+		dst.SetBasenameStr(src.Filename)
```

Comment:
We need to update the line 110, with dst.SetPathnameStr("")

Reply:
I was wondering for this one, I'll make the change